include/mbgl/shaders/mtl/shader_group.hpp

```diff
@@ -57,6 +92,15 @@ class ShaderGroup final : public gfx::ShaderGroup {
         return shader;
     }
 
+protected:
+    static constexpr bool usePermutations() noexcept {
+#if NO_METAL_PERMUTATIONS
```

Comment:
I assume you used this for benchmarking. Does it make sense to keep it in?

Reply:
My only concern is if there are cases where shader compilation time ends up a problem.  But it would be easy enough to put back in